frontend/src/constants/index.js

```diff
@@ -59,6 +59,34 @@ export const pollingOptions = [
   }
 ];
 
+export const logsLimitOptions = [
```

Comment:
A default value of this options will be set `1000`, is it fine? I don't follow (or forget...) a discussion related to this default value.

Reply:
The reason I chose `1000` for the default value of `logs_limit` is:
- For users who have only small logs, they will not be affected
- For users who have huge logs, ChainerUI can somehow handle huge logs when `logs_limit <= 1000`
- For users who feel ChainerUI is slow, they can change `logs_limit` to a smaller value, i.e. `500` or `100`.

I will add a description of changing `logs_limit` to the document.